pkg/preparer/orchestrate_test.go

```diff
@@ -218,9 +244,47 @@ func testPreparer(t *testing.T, f *FakeStore) (*Preparer, *fakeHooks, string) {
 	hooks := &fakeHooks{}
 	p.hooks = hooks
 	p.store = f
+	p.podStatusStore = &FakeStatusStore{}
 	return p, hooks, podRoot
 }
 
+func TestInstallWhiteListPodsInTheIntent(t *testing.T) {
+	whiteListPods := make(map[types.PodID]bool)
+	whiteListPods["hello"] = true
+
+	store := &FakeStore{
+		currentManifest: testManifest(t),
+		wipedReality:    true,
+	}
+
+	p, _, fakePodRoot := testPreparer(t, store)
+	defer p.Close()
+	defer os.RemoveAll(fakePodRoot)
+
+	err := p.InstallWhiteListPods(whiteListPods)
+	Assert(t).IsTrue(err != nil, "Pod cannot be installed.")
+	s := strings.Split(err.Error(), " ")
+	podID := s[len(s)-1]
+	Assert(t).AreEqual(podID, "hello", "The podID to install should be the one in intent")
+}
+
+func TestInstallWhiteListPodsNotInTheIntent(t *testing.T) {
```

Comment:
Is this name implying that this new code change will install pods that are in the whitelist but not in intent?

Reply:
Have refined the names of the tests.